query-engine/dmmf/test_files/functions.json

```diff
@@ -48,7 +48,7 @@
           "default": {
             "name": "cuid",
             "args": [
-              2
+              1
```

Comment:
This is a result of https://github.com/prisma/prisma-engines/pull/5047/commits/9d04afbf7ca0c7c11ea6921262a09cae6604ba08.